keys/employees-keys/README.md

```diff
@@ -3,172 +3,117 @@
+Clone repository:
 
-```
-RSA keys may be between 1024 and 4096 bits long.
-@What keysize do you want? (2048) 4096
-Requested keysize is 4096 bits
+```shell
+git clone https://github.com/3mdeb/3mdeb-secpack.git
 ```
 
-Use a 2 years expiry key and then confirm your choice:
+Change directory:
 
-```
-Please specify how long the key should be valid.
-         0 = key does not expire
-      <n>  = key expires in n days
-      <n>w = key expires in n weeks
-      <n>m = key expires in n months
-      <n>y = key expires in n years
-Key is valid for? (0) 2y
-Key expires at Fri 07 Apr 2023 11:40:33 AM CEST
-Is this correct? (y/N) y
+```shell
+cd 3mdeb-secpack/scripts
 ```
 
-Enter your real name:
+Generate key:
 
-```
-GnuPG needs to construct a user ID to identify your key.
-
-@Real name: Your Name
+```shell
+./gen-new-employee-key.sh "Your Name" "your.name@3mdeb.com"
 ```
 
-Company's email address:
+Please note your key id printed at the end:
 
 ```
-@Email address: your.name@3mdeb.com
-```
-
-Comment field is optional:
+(...)
+[ultimate] (1). Your Name (Employee Cert Key) <your.name@3mdeb.com>
 
-```
-@Comment:
+Your KEY_ID: D9E4EB63705C3897
 ```
 
-Check and confirm your details:
+### Encrypt revocation cert
 
-```
-You selected this USER-ID:
-    "Your Name <your.name@3mdeb.com>"
+Import 3mdeb Founder key:
 
-@Change (N)ame, (C)omment, (E)mail or (O)kay/(Q)uit? o
+```shell
+gpg --import key/owner-key/piotr-krol-key.asc
 ```
 
-Enter passphrase twice (second is confirmation):
+Import your 3mdeb Team Leader or Manager key:
 
+```shell
+gpg --import key/employees-keys/your.tl-or-mgr-name@3mdeb.com
 ```
-You need a Passphrase to protect your secret key.
-```
-
-You might have to spend some time moving your mouse if entropy is low:
 
-```
-We need to generate a lot of random bytes. It is a good idea to perform
-some other action (type on the keyboard, move the mouse, utilize the
-disks) during the prime generation; this gives the random number
-generator a better chance to gain enough entropy.
-gpg: key *D458BD99F75CFACB* marked as ultimately trusted
-gpg: directory '/home/user/.gnupg/openpgp-revocs.d' created
-gpg: revocation certificate stored as '/home/user/.gnupg/openpgp-revocs.d/EB32D0276A3DBF45FC7621A2D458BD99F75CFACB.rev'
-public and secret key created and signed.
-```
+Encrypt your revocation certificate (please use your KEY_ID from previous
+point):
 
-This is how it ends successfully (you might see a message about entropy
-multiple times):
+```shell
+gpg --encrypt -r piotr.krol@3mdeb.com -r your.tl-or-mgr-name@3mdeb.com -o \
+your.name@3mdeb.com.rev.gpg ${HOME}/.gnupg/openpgp-revocs.d/*KEY_ID.rev
 
-```
-pub   rsa4096 2022-12-13 [SC] [expires: 2024-12-12]
-      EB32D0276A3DBF45FC7621A2D458BD99F75CFACB
-uid                      VM User (GPG Identity test creation) <vm_user@exampledomain.com>
-sub   rsa4096 2022-12-13 [E] [expires: 2024-12-12]
 ```
 
-The key ID, in this case, is `D458BD99F75CFACB`.
+Output shouold look as follows:
```

Comment:
Instead of pointing out typos, I have enabled pre-commit in this repo: https://github.com/3mdeb/3mdeb-secpack/pull/102/files

Reply:
@macpijan I added a review to #102

Reply:
@pietrushnic responded there

Reply:
@macpijan IIUC we need to merge #102 to close this conversation?

Reply:
@macpijan ping?